### PR TITLE
September 6 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | a0203f7 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-518.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240906_amplify.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+|             | APIGOV-28742 | **Issue**: The Discovery Agent may fail in cases where an API Management Gateway proxy is published with inbound security incomplete or missing. <br />**Resolution**: A fix was made to the handling of inbound security configuration. |
+
+#### Software AG webMethods Agent
+
+* **Software AG Quota Enforcement** - The Software AG webMethods Agent has enhanced to inclued quota enforcement defined in a product plan, traffic usage and traceability.  For more details, see [Connect Software AG webMethods]([https://docs/connect_software_ag_webmethods/index.html)).
+
+#### Traceable Agent
+
+A new Traceable API Security Agent is now available. This agent enables you to map your runtime traffic against your managed API infrastructure and identify unmanaged APIs. In addition, it allows you to visualize and report on operational and security problems by integrating with Traceable API Security. For more details, see [Connect Traceable]([https://www.npmjs.com/package/@axway/axway/v/3.2.14](https://docs/runtime_security/configure_runtime_compliance_traceable/index.html)).
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240906_amplify.md:81]: Not new, initial release

File: content/en/docs/amplify_relnotes/_index.md [hunk 2]
```diff
@@ -20,6 +20,75 @@ To view the *Administration Release Notes* [click here](https://docs.axway.com/b
...
+|--------------------------------------------|
+| AWS Embedded Service                       |
+| Apigee X Embedded Service                  |
+| GitHub Embedded Service                    |
+| Azure Embedded Service                     |
+| SwaggerHub Embedded Service                |
+
+| Runtime Compliance agents                  | Latest on-premise agent version <br />(based on Amplify Agents SDK v1.1.95)  | Minimum supported agent version <br />(based on Amplify Agents SDK v1.1.83)  |
+|--------------------------------------------|--------------------------|------------------|
+| Graylog API Security                       | 1.1.1                   | 1.0.5                           |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/_index.md:53]: Add Traceable here?

File: content/en/docs/amplify_relnotes/20240906_amplify.md (hunk 1 above)
Comment (jcollins-axway) [reply content/en/docs/amplify_relnotes/20240906_amplify.md:81]: I read it as a new, as an update, but it is good

File: content/en/docs/amplify_relnotes/_index.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/_index.md:53]: Added Traceable row to the Index page.   Lisa, you normally update the version numbers on the index page right?

**Review by georgiana-caras — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
I approve

**Pull request merged**